src/data/default_run.yaml

```
@@ -0,0 +1,155 @@
+vector_stores:
+  # LCORE-1498: Disables Llama Stack RAG annotation generation
+  # causing unwanted citation/file markers in model output.
+  annotation_prompt_params:
+    enable_annotations: false
+  default_provider_id: faiss
+  default_embedding_model: # Define the default embedding model for RAG
+    provider_id: sentence-transformers
+    model_id: nomic-ai/nomic-embed-text-v1.5
+safety:
+  default_shield_id: llama-guard
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)